### ISSUE TITLE
What is the criteria for becoming a sponsor of the site?

### ISSUE THREAD

**Issue by 3noch** [haskell-infra/hl#181]
Is there an official criteria for becoming a sponsor? I'm thinking it would involve donations and/or services of some sort?

**Comment by gbaz** [comment]
We don't have a page with official criteria at this point. You're correct that donations in kind or financial support have been what's determined it, but there's no single place where we lay out a set of rules. In general it has been everyone providing in-kind donations, and then the addition of HSOC sponsors. But this is a good question! Are you interested in sponsorship, or just looking to clarify things? cc @ryantrinkle

**Comment by 3noch** [comment]
Yes I'm interested in sponsoring with either/both donations and services. I am part owner of an online marketing company where we use Haskell almost exclusively and would love to contribute free online marketing services to the Haskell ecosystem.

**Comment by gbaz** [comment]
Ok, I'll leave this ticket open because we should document this better in general, but we'll take the specific convo to email.

**Comment by 3noch** [comment]
@gbaz Sounds great.

**Comment by thoughtpolice** [comment]
@3noch Yeah, so just for clarity - when I originally wrote that list, it was because everyone on there essentially offers us some kind of their service, pro-bono (hosting, storage, monitoring), or was a substantial help in recent memory (such as Galois).

In general I don't think I ever put much more thought into it than that -- and all the people there stand out on my mind as those who have provided a lot in the past few years (I mean, if you go back to like the 90s you could even include Yale, but they're way out of that game now).

Finally, thank you for the offering! Help is always appreciated. @gbaz - please CC me on anything if you'd like.
